cli/dcoscli/marathon/main.py

```diff
@@ -860,6 +865,32 @@ def task_stop(self, task_id, wipe):
         emitter.publish(task)
         return 0
 
+    def task_kill(self, task_ids, scale, wipe):
+        """Kill one or multiple Marathon tasks
+
+        :param task_ids: the id of the task
+        :type task_ids: [str]
+        :param scale: Scale the app down after killing the specified tasks
+        :type scale: bool
+        :param wipe: whether remove reservations and persistent volumes.
+        :type wipe: bool
+        :returns: process return code
+        :rtype: int
+        """
+
+        if not task_ids:
+            raise DCOSException('Please provide at least one task ID')
+
+        client = self._create_marathon_client()
+        payload = client.kill_and_scale_tasks(task_ids, scale, wipe)
+
+        emitter.publish(payload)
+        # No deployment was started and no tasks were killed
```

Comment:
You could ask from *all* tasks running and compare but that would have a negative performance impact. The endpoint is built the way it would tell you afterwards whether or not the task was killed - I think we should keep it this way.